app/views/admin/meetings/_invitation_management.html.haml

```diff
@@ -1,34 +1,34 @@
 - if @invitations.any?
   .row
-    .large-12.columns
+    .col
       .panel
         %p
           <strong>#{@invitations.count}</strong> members have RSVP'd to this event.
 
-  .row
-    .large-9.columns
-      %small
+  %div
+    = simple_form_for :meeting_invitations, url: admin_meeting_invitations_path do |f|
       .row
-        = simple_form_for :meeting_invitations, url: admin_meeting_invitations_path do |f|
-          .medium-6.columns
-            = f.select :member,
-                       Member.all.map { |u| ["#{u.full_name}", u.id] },
-                       { include_blank: true }, { class: 'chosen-select', required: true,
-                       data: { placeholder: t('messages.invitations.select_a_member_to_rsvp') } }
-            = f.input :meeting_id, as: :hidden, input_html: { value: @meeting.slug }
+        .col-6
+          = f.select :member,
+                      Member.all.map { |u| ["#{u.full_name}", u.id] },
+                      { include_blank: true }, { class: 'chosen-select', required: true,
+                      data: { placeholder: t('messages.invitations.select_a_member_to_rsvp') } }
+          = f.input :meeting_id, as: :hidden, input_html: { value: @meeting.slug }
 
-          .large-3.columns
-            %span{ 'data-tooltip': true, 'aria-haspopup': 'true', class: 'has-tip', title: t('admin.workshop.manage_rsvps.text') }
-              %i.fas.fa-info
-            = f.submit 'Add'
-      %br
+        .col-3
+          %span{ 'data-tooltip': true, 'aria-haspopup': 'true', class: 'has-tip', title: t('admin.workshop.manage_rsvps.text') }
+            %i.fas.fa-info
+          = f.submit 'Add'
```

Comment:
@KimberleyCook would be good to style this button, at least make it a `f.button :button` element